source/_patterns/02-organisms/components/content-aside~status-title-long.json

```diff
@@ -0,0 +1,48 @@
+{
+  "status": {
+    "title": "Author Accepted Manuscript",
```

Comment:
The logic in patterns-php should set the titleLength to long if above 23 characters.